lib/Raven/Serializer.php

```diff
@@ -109,6 +109,12 @@ protected function serializeValue($value)
         if (is_null($value) || is_bool($value) || is_float($value) || is_integer($value)) {
             return $value;
         } elseif (is_object($value) || gettype($value) == 'object') {
+            // If the value is a callable and expects no parameters, we evaluate the
+            // callable value by calling it.
+            if (is_callable($value) && Raven_Util::getCallableParamNum($value) === 0) {
+                return $value();
```

Comment:
What if people starts complaining because closures are called and invokable classes not? By the way, these points is still valid:

> you can't base your assumptions on the fact that any callable/method should be called